conftest.py

```diff
@@ -73,3 +76,76 @@ def undo(self):
 @pytest.fixture(autouse=True)
 def default_patches(request):
     return DefaultPatches(request)
+
+
+@pytest.fixture()
+def set_env_variables():
+    os.environ["BORG_CHECK_I_KNOW_WHAT_I_AM_DOING"] = "YES"
+    os.environ["BORG_DELETE_I_KNOW_WHAT_I_AM_DOING"] = "YES"
+    os.environ["BORG_PASSPHRASE"] = "waytooeasyonlyfortests"
+    os.environ["BORG_SELFTEST"] = "disabled"
+
+
+class ArchiverSetup:
+    EXE: str = None  # python source based
+    FORK_DEFAULT = False
+    BORG_EXES = []
+
+    def __init__(self):
+        self.archiver = None
+        self.prefix = ""
```

Comment:
A couple tests used it:

https://github.com/borgbackup/borg/blob/33645ad38a312662147a4f72d544fa25e0cff71c/src/borg/testsuite/archiver/rcreate_cmd.py#L26

https://github.com/borgbackup/borg/blob/33645ad38a312662147a4f72d544fa25e0cff71c/src/borg/testsuite/archiver/checks.py#L230

However, I could remove prefix from `archiver`, set it only in `remote_archiver`, and then just replace these instances of `prefix` with a `hasattr(archiver, 'prefix')` check first. Just wasn't sure if this would make the code more difficult to read.

Reply:
Maybe it is cleaner to set `self.prefix = None` here.